style_editor.html

```diff
@@ -7213,7 +7221,7 @@
     super("Mix functions", ARGS);
     this.LAYERS = Array.from(ARGS);
     for (var i = 1; i < this.LAYERS.length + 1; i++)
-      this.add_arg("FUNCTION" + i, "FUNCTION", "COLOR " + i);
+      this.add_arg("FUNCTION" + i, "FUNCTION", "FUNCTION " + i);
```

Comment:
Found this. The functions in the LayeredFunctions list as COLOR 1, COLOR 2, COLOR 3 in the Structured view.

Reply:
nice find (but mixing cleanup and bugfixes is not generally recommended)